src/styles/card.scss

```diff
@@ -0,0 +1,2 @@
+@import url(./header.scss);
```

Comment:
what this file is doing?
move all card styles here instead and delete `product_card.scss`